src/main/java/cdw/springtraining/gatekeeper/security/JwtTokenProvider.java

```diff
@@ -0,0 +1,82 @@
+package cdw.springtraining.gatekeeper.security;
+
+import io.jsonwebtoken.*;
```

Comment:
import only needed package and avoid importing all packages

Reply:
Removed wildcard entries